platform/ui/src/components/SplitButton/SplitButton.tsx

```diff
@@ -31,11 +29,17 @@ const classes = {
   Primary: ({ primary, isExpanded }) =>
     classNames(
       baseClasses.Primary,
-      primary.isActive && !isExpanded
-        ? 'bg-primary-light rounded-tr-md rounded-br-md active'
+      primary.isActive
```

Comment:
The className for the primary button is calculated here and passed down to the button as a prop.

Reply:
This is getting quite ugly - this needs some way to document the behaviour and why the various behaviour is as it is so that in the future people can figure out how to fix bugs in it.

Reply:
I agree it is quite ugly, but I would say it was not much prettier before. The intended behaviour is described by the OHIF specs laid out in Zeplin and this code fulfills that behaviour. Is that document enough to guide bug fixing?

Unfortunately the complexity is a result of how intertwined the look-and-feel behaviour of the two buttons have been defined in the OHIF specs. To be honest here, my past dealings with split buttons is that in essence the two buttons are separate buttons when it comes to look and feel. A quick look around at bootstrap and React MUI (https://mui.com/material-ui/react-button-group/#split-button) further confirms this. For example, I would say in general it is very rare for a split button that hovering over one of the two buttons affects the look of the other. However, the OHIF spec gives the primary button rounded corners on the left and square corners on the right when hovering over the secondary, but the primary has rounded corners when active. Now that my change allows for a separate button to be the primary it makes the logic here difficult, but doable.

If the two buttons were a bit more independent look-and-feel wise, this ugliness would go away. Thoughts?

Reply:
Can you create an issue to separate out the buttons?  I think that would definitely make it cleaner overall - it was ugly before and is just as ugly now.

Reply:
I am not sure that a simple issue would do here. Separating the buttons amounts to a change in design/spec. I am not sure we want to change the spec any time soon. Thoughts @sedghi ?